### PR TITLE
chore: Test noop nodelist method

### PR DESCRIPTION
**Pull request by He-Pin**
Motivation:
Show why https://github.com/json-everything/json-everything/pull/831 is needed

### CODE REVIEW

**Comment by He-Pin** [comment]
@gregsdennis ping~

File: src/JsonPath.Tests/ExpressionTests.cs [hunk 1]
```diff
@@ -53,7 +53,7 @@ public void ExpressionWithNoOpFunctionWorks()
 		FunctionRepository.RegisterNodelistFunction<NoOpFunction>();
 
 		// should do the same thing as $[?@.*]
-		var path = JsonPath.Parse("$[?noop(@.*)]");
```
Comment (gregsdennis) [inline src/JsonPath.Tests/ExpressionTests.cs:56]: Don't change existing tests.  Create new tests.

**Comment by gregsdennis** [comment]
This is your new PR.

Steps are:

1. Create an issue so changes can be discussed (you didn't do this)
2. Once discussed, create a PR with a test that illustrates the problem.  The build should fail for this.
3. Apply one or more commits that fix the problem.  The build passes.

File: src/JsonPath.Tests/ExpressionTests.cs (hunk 1 above)
Comment (He-Pin) [reply src/JsonPath.Tests/ExpressionTests.cs:56]: Have some network issue (GFW), will update later.

**Comment by gregsdennis** [comment]
I don't need to be pinged.

**Pull request closed without merging**